### PR TITLE
Automated cherry pick of #41: Update release branch regex

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #41 on release-calient-v3.9.

#41: Update release branch regex

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
